### ISSUE TITLE
[Question] What's the motivation behind adversarial loss: ReLU(1 + x) + ReLU(1 - x)

### ISSUE THREAD

**Issue by Luciennnnnnn** [G-U-N/Phased-Consistency-Model#15]
Hi, what's the motivation behind adversarial loss: ReLU(1 + x) + ReLU(1 - x), is there any reference?

**Comment by G-U-N** [comment]
This is a typo here, and it should be $\textrm{ReLU}(1 + D (\tilde{x}_s, c, s)) + \textrm{ReLU}(1 - D(\hat{x}_s, c, s))$, where $D$ is the discriminator, $c$ is the text conditions and $s$ is the timestep conditions. The $\textrm {ReLU}$ function is equivalent to normally applied hinge loss.

**Comment by Luciennnnnnn** [comment]
yes, thank you for clarify. What I'm asking is the name/origin of this adversarial loss. So it is hinge loss, got it!
